src/DI/ServiceProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types = 1);
+
+namespace BrandEmbassy\Slim\DI;
+
+use Nette\DI\Container;
+use Nette\DI\MissingServiceException;
+
+final class ServiceProvider
+{
+    /**
+     * @return object
+     */
+    public static function getService(Container $container, string $serviceName)
```

Comment:
I would rename the second argument because it's not saying it will also find by type if service with this name does not exist. Maybe just service?